src/mixins/auth-api-mixin.ts

```diff
@@ -55,7 +55,11 @@ export default class AuthApiMixin extends Vue {
       .then(response => {
         if (response?.data) {
           return {
-            contacts: response.data.contacts,
+            contact: {
+              email: response.data.contacts[0].email,
+              phone: response.data.contacts[0].phone,
+              extension: response.data.contacts[0].phoneExtension
+            },
```

Comment:
Where we used this data, it was de-structured down to the first contact item in the array.
This serves two purposes
- to clean up that de-structuring and only do it in one place
- to map the `phoneExtension` property to `extension` property. 

Lear (and locally here) we referred to `extension` property but there is a disconnect from auths `phoneExtension` property name.
I'm not sure when the values name in auth changed or if ever had, but the display of the extension value was broken due to this.